packages/fxa-auth-server/lib/senders/emails/partials/automatedEmailInactiveAccount/en.ftl

```diff
@@ -0,0 +1 @@
+automated-email-inactive-account = This is an automated email. You are receiving it because you have a Mozilla account and it has been 2 years since your last sign-in.
```

Comment:
We should use the `-product-mozilla-account` term here however. (Unless there's a technical reason not to.)